src/__tests__/render.test.js

```diff
@@ -378,3 +378,23 @@ test('renders options.wrapper around updated node', () => {
     </RCTSafeAreaView>
   `);
 });
+
+test('returns container', () => {
+  const { container } = render(<View testID="inner" />);
+
+  expect(container).toBeDefined();
+  expect(container.type).toBe(View);
+});
+
+test('returns wrapped component as container', () => {
+  const WrapperComponent = ({ children }) => (
+    <SafeAreaView testID="wrapper">{children}</SafeAreaView>
+  );
+
+  const { container } = render(<View testID="inner" />, {
+    wrapper: WrapperComponent,
+  });
+
+  expect(container).toBeDefined();
+  expect(container.type).toBe(WrapperComponent);
```

Comment:
Consider adding testID check. Currently there is no reason for testID props to be present.